Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:2.5.1-alpine3.7
+
+WORKDIR /app
+
+COPY Gemfile Gemfile.lock ./
+RUN apk --no-cache add alpine-sdk && bundle install && apk del alpine-sdk
+
+EXPOSE 9292
```

Comment:
whats the symbol next to the port 9292?

Reply:
It's github warning that there's no line break at the end of the file.